Gemfile

```diff
@@ -12,44 +12,36 @@ gem "jekyll", "~> 4.1"
 
 gem "jekyll-theme-hydejack", "~> 9.1"
 
-# IMPORTANT: The followign gem is used to compile math formulas to 
+# IMPORTANT: The followign gem is used to compile math formulas to
 # KaTeX during site building.
 #
 # There are a couple of things to know about this gem:
-# *  It is not supported on GitHub Pages. 
+# *  It is not supported on GitHub Pages.
 #    You have to build the site on your machine before uploading to GitHub,
 #    or use a more permissive cloud building tool such as Netlify.
 # *  You need some kind of JavaScript runtime on your machine.
-#    Usually installing NodeJS will suffice. 
+#    Usually installing NodeJS will suffice.
 #    For details, see <https://github.com/kramdown/math-katex#documentation>
 #
 # If you're using the MathJax math engine instead, free to remove the line below:
 gem "kramdown-math-katex"
 
-# A JavaScript runtime for ruby that helps with running the katex gem above.
-gem "duktape"
-
-# Fixes `jekyll serve` in ruby 3
-gem "webrick"
```

Comment:
not sure if this is needed or not. Mine has this but I found another online that does not